model/app.py

```diff
@@ -26,15 +27,15 @@
 from model.messages import GeneToReactionsRemote, GeneRequest
 
 
-ORGANISMS = {
-    'iJO1366',
-    'iMM904',
-    'iMM1415',
-    'iNJ661',
-    'iJN746',
-    'e_coli_core',
+ORGANISM_TO_MODEL = {
```

Comment:
duplicates now with https://github.com/DD-DeCaF/iloop-to-model/blob/master/iloop_to_model/settings.py#L5 and https://github.com/DD-DeCaF/iloop-to-model/blob/master/iloop_to_model/app.py#L27
reasoning for having this call in `iloop-to-model` was so that `model` service wouldn't be familiar with the concept of organism and about the iloop organism naming conventions since organisms names are not used by model service anywhere
and it leads to extra call for `iloop-to-model` service for figuring out the short code https://github.com/DD-DeCaF/iloop-to-model/blob/24a477af7e9663f3b66e87969aaae7399b701a1e/iloop_to_model/iloop_to_model.py#L108